user-service/src/main/java/rs/edu/raf/banka1/repositories/CapitalRepository.java

```diff
@@ -13,7 +13,6 @@
 @Repository
 public interface CapitalRepository extends JpaRepository<Capital, Long> {
 
-    Optional<Capital> getCapitalByCurrency_CurrencyCode(String curr);
     Optional<Capital> getCapitalByListingIdAndListingType(Long listingId, ListingType listingType);
     List<Capital> getCapitalsByBankAccountAndListingType(BankAccount bankAccount, ListingType listingType);
```

Comment:
Ovo može da se izbriše, pošto nemamo u tabeli red gde će i `bankAccount` i `listingType` biti != null, pa nam je ovaj query suvišan.